lib/api/src/grpc/proto/points.proto

```diff
@@ -335,20 +352,21 @@ message ScoredPoint {
   optional Vectors vectors = 6; // Vectors to search
 }
 
-message GroupId {
-  oneof kind {
-    // Represents a double value.
-    uint64 unsigned_value = 1;
-    // Represents an integer value
-    int64 integer_value = 2;
-    // Represents a string value.
-    string string_value = 3;
+message PointGroup {
+  message GroupId {
```

Comment:
I am not entirely sure this change is backward compatible.
It would be great if you could confirm that it won't break existing clients.